packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -2,46 +2,72 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:mustache/mustache.dart' as mustache;
 import 'package:yaml/yaml.dart';
 
 import 'base/file_system.dart';
 import 'dart/package_map.dart';
 import 'globals.dart';
 
-dynamic _loadYamlFile(String path) {
-  if (!fs.isFileSync(path))
+class Plugin {
+  final String name;
+  final String path;
+  final int version;
```

Comment:
This was really just future-proofing. All plugins are currently on version 1 of the plugin interface, so there was no need to check.

Removed for now, to avoid confusing things. We can always add the version, and treat anything with a missing version tag as version 1.